Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -91,16 +92,18 @@ public static bool IsAzureEnvironment()
         /// It is written to require least dependencies but framework assemblies.
         /// Caller is responsible for synchronization.
         /// </summary>
+        [SuppressMessage("Microsoft.Usage", "CA1801:Parameter 'siteRestrictedJwt' is never used", 
+            Justification = "Method signature has to be the same because it's called via reflections from web-deploy")]
         public static async Task Run(string requestId, string siteRestrictedJwt, TraceListener tracer)
```

Comment:
I wonder if we could introduce Run2(string requestId, TraceListener tracer) and have this Run calling it.   Once we do that, MSDeploy can switch to call Run2 and we can get rid of the Run.    Open for better name than Run2.

Reply:
how about `RunPostDeployment()` or `Invoke()`? any preference?

Reply:
or `Run2()` if you don't like either one

Reply:
`Invoke()` it is